app/js/handlebarsUserTable.js

```diff
@@ -19,10 +19,18 @@ function fillUserTable(HBTemplate) {
                     let html = template(userData)
                     user_list.innerHTML += html
                 })
+                let userItems = document.querySelectorAll(".user_item")
+                userItems.forEach(function (userItem) {
+                    userItem.addEventListener('click', function () {
+                        console.log(userItem)
+                    })
+                })
             } else {
                 user_list.innerHTML = "Please contact Admin, user list unavailable"
             }
         })
+
```

Comment:
empty space --> is this required?

take it out if not.

Reply:
readability